app/lib/statuses-map.js

```diff
@@ -0,0 +1,7 @@
+export default {
```

Comment:
Yarp, I saw there was a conflict when I rebased master with a similar file so I created a new one as a placeholder and am gonna fix it at the end